pypechain/main.py

```diff
@@ -11,7 +11,7 @@
 from typing import NamedTuple, Sequence
 
 from pypechain.render.init import render_init_file
-from pypechain.render.main import render_contracts
+from pypechain.render.render import render_contracts
```

Comment:
nit: adding from .render import render_contracts to the init in the render folder will remove the need for double .render.render

nbd if you prefer to use the full path I think it makes things neater to remove that lowest level when there's a single "main" type of file